test/test_basic.py

```diff
@@ -14,6 +14,13 @@
 os.environ.pop('SSH_AUTH_SOCK', None)
 os.environ.pop('SSH_ASKPASS', None)
 
+# some host key which isn't the one from mock-ssh
+NONMATCHING_HOSTKEY = (
```

Comment:
I feel like maybe this should be in a file.

At the same time, I can't find the officially suggested way to do that with pytest, and I don't really feel like forcing you to do another round...